config/redis.yml

```diff
@@ -14,6 +14,9 @@ development: &defaults
     namespace: user_identities
     each_ttl: 1800
   # DO NOT CHANGE ABOVE TTL
+  identity_store:
+    namespace: identity-store
+    each_ttl: 2592000
```

Comment:
I would bound this much more tightly, keeping it close to the maximum time that it would live given Sidekiq retries (a.k.a, if you have 10-12 Sidekiq retries, I believe the 12'th retry would be something like 24 hours from the first, so the ttl should be roughly 24 hours)